src/components/HomepageFeatures/index.tsx

```diff
@@ -74,25 +82,27 @@ const featureList: FeatureItem[] = [
 
 export default function HomepageFeatures(): JSX.Element {
   const list = featureList.map((listItem) => (
-    <div key={uuidv4()}>
-      <UnorderedList>
-        <UnorderedList.Item>
-          <Heading size="level-5">
-            <Link to={listItem.to}>{listItem.title}</Link>
-          </Heading>
-          <Paragraph>{listItem.description}</Paragraph>
-        </UnorderedList.Item>
-      </UnorderedList>
-    </div>
+    <Grid.Cell key={uuidv4()} span={4}>
+      <Card>
+        {typeof listItem.image === 'string' ? (
```

Comment:
Isn't it possible to import the other SVG icons in the same way as the GitHub icon? Then you don't need this conditional and it will import SVGs natively, than as image elements.

Reply:
Done this